vendor.conf

```diff
@@ -32,7 +32,7 @@ github.com/dustin/go-humanize 8929fe90cee4b2cb9deb468b51fb34eba64d1bf0
 github.com/golang/mock bd3c8e81be01eef76d4b503f5e687d2d1354d2d9
 github.com/google/certificate-transparency 0f6e3d1d1ba4d03fdaab7cd716f36255c2e48341
 github.com/hashicorp/go-immutable-radix 8e8ed81f8f0bf1bdd829593fdd5c29922c1ea990
-github.com/hashicorp/go-memdb 98f52f52d7a476958fa9da671354d270c50661a7
+github.com/hashicorp/go-memdb 608dda3b1410a73eaf3ac8b517c9ae7ebab6aa87 https://github.com/floridoo/go-memdb
```

Comment:
Our vendored version is a few days newer, but I checked and the only changes that the fork misses are new features that we don't use. There are no missing bugfixes.